README.md

```diff
@@ -65,7 +65,7 @@ This directory should contain the following files and directories, where `<slug>
 * *optional*: `example/*.ceylon`
   * Other files may be included in the `example/` directory if appropriate.
 
-As a reminder, the README does not need to be added to this track - it is automatically created using the data in [x-common](https://github.com/exercism/x-common).
```

Comment:
This becomes false when https://github.com/exercism/meta/issues/15 happens, but one step at a time. Update the README when it happens.